tools/introspection/source/introspection_app.cpp

```diff
@@ -701,43 +700,26 @@ void IntrospectionApp::runIntrospection(const iox::units::Duration updatePeriodM
         {
             prettyPrint("### MemPool Status ###\n\n", PrettyOptions::highlight);
 
-            std::deque<const MemPoolIntrospectionTopic*> mempoolSamples;
             const void* rawMempoolSample{nullptr};
 
+            while (!rawMempoolSample)
```

Comment:
braces? @elBoberido  why didn't you see that?

Reply:
Oh no, shame on me!